requirements.txt

```diff
@@ -0,0 +1 @@
+Flask 
```

Comment:
Please pin the Flask version

Reply:
just to make sure you mean to hard code it in the YML file?

Reply:
No. Flask is a versioned python module. Your code can work with the latest version of Flask today. But, might not work with future Flask versions. No one can guarantee backward compatibility. To avoid this situation, it is advised to pin the version you worked with when implemented it. For additional information: https://pip.pypa.io/en/stable/user_guide/#pinned-version-numbers

Reply:
ack! 
done in the new commit.